sys/include/net/fib.h

```diff
@@ -44,7 +44,8 @@ typedef struct rp_address_msg_t {
 } rp_address_msg_t;
 
 /**
- * @brief message type for RP notification: unreachable destination
+ * @brief message type for RP notification: no route to destination available,
+          start route discovery if appropriate
```

Comment:
just for consistency reasons, could you add an asterisk at the beginning of this line?